static/css/components/nav-bar.less

```diff
@@ -40,7 +40,7 @@
 
 .nav-bar-wrapper {
   display: flex;
-  align-items: baseline;
+  align-items: center;
```

Comment:
Nice good find! The previous solution and `align-items: center` have a bad side-effect of squishing the click region of the arrow to be very small, which is bad for accessibility on touch screens. This will stretch it out to be larger, and seems to look the same visually

```suggestion
  align-items: stretch;
```

But actually, to fix the centering issue and the clipping issue, giving the `button.nav-arrow` element styling like this:

```css
.nav-arrow {
  position: absolute;
  right: 0;
  top: 0;
  bottom: 0;
  width: 44px;
  background: linear-gradient(to right, transparent, white 10px);
  z-index: 10000;
  padding: 0;
}
```

Should do the trick! Eg 
![image](https://github.com/user-attachments/assets/e47f1066-27bd-4ae1-be1a-99fcee661edb)